docs/cody/clients/install-jetbrains.mdx

```diff
@@ -151,7 +151,7 @@ Like the Edit Code and Generate Unit Test commands, you can generate inline docu
 
 ## Supported LLM models
 
-Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Both Cody Free and Pro users can choose from a list of supported LLM models for Chat and Commands.
+Users on Cody **Free**, **Pro**, and **Enterprise** can choose from a list of supported LLM models for Chat and Commands. Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands.
```

Comment:
Isn't it Claude Sonnet 3.5?

Reply:
![image](https://github.com/user-attachments/assets/e6e968e2-6ebe-4f72-8b74-45c4eed8660b)